--- origsrc/xman/configure.ac	2008-11-13 12:03:03.000000000 -0600
+++ src/xman/configure.ac	2008-11-13 15:19:16.653992400 -0600
@@ -90,7 +90,7 @@
 	*darwin* | *openbsd* | *netbsd* )
 	    MAN_CONFIG_STYLE="OpenBSD"
 	    ;;
-    	*linux* )
+    	*linux* | *cygwin* )
 	    MAN_CONFIG_STYLE="Linux"
 	    ;;
         *bsd* )
--- origsrc/xman/vendor.h	2008-11-13 12:03:03.000000000 -0600
+++ src/xman/vendor.h	2008-11-13 15:33:06.638367400 -0600
@@ -59,7 +59,7 @@
 #  define SEARCHDIR  MAN
 #endif
 
-#if ( defined(sgi) || (defined(i386) && (defined(SYSV) || defined(SVR4))  && !defined(sun)) || (defined(BSD) && (BSD >= 199103)) || defined(linux) )
+#if ( defined(sgi) || (defined(i386) && (defined(SYSV) || defined(SVR4))  && !defined(sun)) || (defined(BSD) && (BSD >= 199103)) || defined(linux) || defined(__CYGWIN__) )
 # define SEARCHOTHER CAT
 #endif
 
@@ -179,7 +179,7 @@
 #  define FORMAT "| eqn | tbl | nroff -mandoc"
 # elif defined(BSD) && (BSD >= 199103)
 #  define FORMAT "| eqn | tbl | nroff -man"
-# elif defined(linux)
+# elif defined(linux) || defined(__CYGWIN__)
 #  define FORMAT "| pic | eqn | tbl -Tlatin1 | GROFF_NO_SGR= groff -Tlatin1 -mandoc"
 # else
 #  define FORMAT "| neqn | nroff -man"      /* The format command. */
